### ISSUE TITLE
Is `set-url-from-bookmark-new-buffer` gone?

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1482]
Hi,

Nyxt version-2-pre-release-7 includes the command `set-url-from-bookmark-new-buffer` (C-m C-o in default). This is one of the bindings I use the most.

Unfortunately, the current version on master (version 2.0.0-20-g664d8953) does **not** include this command. 

I can only find `set-url-from-bookmark` (with no new-buffer).

Is this intentional?

I thought it could be a bug. I miss this.

The details of my set-up (OS, graphic drivers and et cetera) were provided [here](https://github.com/atlas-engineer/nyxt/issues/1412).

**Comment by aartaka** [comment]
It is intentional. What you can do is `set-url-from-bookmark` -> `M-return` (in Emacs keybindings) -> Choose `NEW-BUFFER-LOAD`. 

If you want to have it as a separate command, you can define a copy of `set-url-from-bookmark` in your initfile, with just an order of `:actions` swapped.

The rationale behind removal was that we have moved to the `prompter` API that allows us to list any number of additional actions for a command, instead of defining additional commands with slightly different actions. Thus, there's only one `set-url-from-bookmark`, but it has two actions.

**Comment by pdelfino** [comment]
@aartaka , thanks for the explanation and advice! I am using `set-url-from-bookmark` -> `M-return ` -> Choose `NEW-BUFFER-LOAD`.

I would like to use this opportunity to ask a question based on your previous answer. 

I am running Nyxt from source and with the REPL open by its side while I am web surfing. I noticed that this additional actions you mentioned are **not** shown as keyloggs in the REPL.

For instance, my REPL shows:

```
<INFO> [10:53:37] Pressed keys: C-m
<INFO> [10:53:44] Loading "https://blog.samaltman.com/".
<INFO> [10:53:45] Finished loading "https://blog.samaltman.com/".
```

And I did `C-m o` (highlight the `o`) plus the `additional actions` you recommended. Wouldn't it be better to see them in the REPL? 

I think this could help catching possible bugs in the future. But I might be naive about this. Curious about your thoughts.
